### DISCUSSION TITLE
v2.1.5

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#49] [Announcements]
- New provider gogoanime(consumet) #21 
- Mx Player pro support #42 
- Persist View type #41 
- UI changes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.1.5'>v2.1.5</a>.</em>

**Comment by Mope04** [comment]
Discord or telegram when?🙏

**Reply by Zenda-Cross** [reply]
waiting for to complete 100 stars

**Reply by Zenda-Cross** [reply]
created it any way https://discord.gg/cr42m6maWy
